db/schema.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE pet_types (
+  id SERIAL PRIMARY KEY,
+  type VARCHAR NOT NULL,
+  description TEXT

```

Comment:
What does TEXT do?

Reply:
TEXT is like VARCHAR except it doesn't have a character limit.